Gruntfile.js

```diff
@@ -181,6 +181,9 @@ module.exports = function(grunt) {
              ' && curl -X DELETE http://admin:pass@localhost:5984/medic-audit-test' +
              ' && kanso push http://admin:pass@localhost:5984/medic-test'
       },
+      test_integration_setup: {
+        cmd: 'npm install api/',
```

Comment:
I have to admit, this kind of makes my eye twitch. I don't know enough about NPM to understand the implications of this I have to be honest. Is it fine? Is it basically just `cp -r api/node_modules ./node_modules`? If we define two different versions for the same dependency in webapp and api will they co-exist correctly?